published-namespaces.txt

```diff
@@ -445,6 +445,10 @@ Google.Ads
 Google.Ads.Identifier
 Google.Ads.InteractiveMedia.PAL
 Google.Ads.InteractiveMedia.PAL.Utils
+Google.Ads.InteractiveMedia.V3.API
```

Comment:
Looking through this file, every other namespace we have used "Api" instead of "API", so let's stay consistent with "Api".

Reply:
I allways forget. In some repos we used JVM/API and in other Jvm/Api.

Reply:
fixed